### PR TITLE
Storage Boosting2.0

### PR DESCRIPTION
**Pull request by dev-j04**
- Apply Boosting 2.0 to StorageV21
- Add Boosting 2.0 test
- Update to typescript

### CODE REVIEW

**Review by sybil-attack — DISMISSED** [review]

File: strategies/low_risk/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+*node_modules
```
Comment (bolidetech) [inline strategies/low_risk/.gitignore]: what the point of adding .gitignore to this directory if we already have one in the root?

File: strategies/low_risk/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{
```
Comment (bolidetech) [inline strategies/low_risk/tsconfig.json:1]: could we put this config to the root directory?

Comment (dev-j04) [reply strategies/low_risk/tsconfig.json:1]: Typescript is running in /strategies/low_risk only.
We don't need to move that config to root folder.
If we move this to root folder, "includes", "files" options should indicate "/strategies/low_risk". but other projects don't use this config.

File: strategies/low_risk/.gitignore (hunk 1 above)
Comment (dev-j04) [reply strategies/low_risk/.gitignore]: fixed

**Review by bolidetech — APPROVED** [review]

**Pull request merged**